Video-Transformers/Podfile

```diff
@@ -0,0 +1,8 @@
+require_relative '../OpenTokSDKVersion'
+
+platform :ios, '13'
+use_frameworks!
+
+target 'Video-Transformers' do
+  pod 'VonageClientSDKVideo', '2.25.2'
```

Comment:
Is the pod name ok.. I am not seeing it on cocoapods. also if correct I hope it is XC framework.

Reply:
@chetanvangadiTokbox any idea why VonageClientSDKVideo does not appear on cocoapods? It works well running pod install, but it seems it is not present visually on the website